### PR TITLE
add version.rb and replace Gem.loaded_specs

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

**Comment by blowmage** [comment]
Rebase?

File: google-cloud-dialogflow/lib/google/cloud/dialogflow/version.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Google LLC
```
Comment (blowmage) [inline google-cloud-dialogflow/lib/google/cloud/dialogflow/version.rb]: Is this year supposed to be 2017?

File: google-cloud-speech/lib/google/cloud/speech/v1/cloud_speech_pb.rb [hunk 2]
```diff
@@ -55,6 +54,47 @@
     value :OGG_OPUS, 6
     value :SPEEX_WITH_HEADER_BYTE, 7
   end
+  add_message "google.cloud.speech.v1.RecognitionMetadata" do
```
Comment (blowmage) [inline google-cloud-speech/lib/google/cloud/speech/v1/cloud_speech_pb.rb]: Is there a way to add the version file without also adding these other changes?

File: google-cloud-talent/lib/google/cloud/talent/v4beta1/resume_service_client.rb [hunk 3]
```diff
@@ -27,6 +27,8 @@
 
 require "google/cloud/talent/v4beta1/resume_service_pb"
 require "google/cloud/talent/v4beta1/credentials"
+require "google/cloud/talent/version"
+require "google/cloud/talent/version"
```
Comment (blowmage) [inline google-cloud-talent/lib/google/cloud/talent/v4beta1/resume_service_client.rb]: The line is added twice here.

**Comment by TheRoyalTnetennba** [comment]
Going to re-open in new PR without running synthtool

**Pull request closed without merging**